src/jdk.jlink/share/classes/jdk/tools/jmod/JmodOutputStream.java

```diff
@@ -55,16 +55,17 @@ class JmodOutputStream extends OutputStream implements AutoCloseable {
      * This method creates (or overrides, if exists) the JMOD file,
      * returning the output stream to write to the JMOD file.
      */
-    static JmodOutputStream newOutputStream(Path file, LocalDateTime date) throws IOException {
+    static JmodOutputStream newOutputStream(Path file, LocalDateTime date, int compLevel) throws IOException {
```

Comment:
Suggest to name this parameter  `compressLevel` to make it clear.

Reply:
Sure, renamed.